packages/runtime/src/editor-preview.main.ts

```diff
@@ -66,6 +66,11 @@ type ToddlePreviewEvent =
       type: 'style_variant_changed'
       variantIndex: number | null
     }
+  | {
+      type: 'global_formulas'
```

Comment:
Should we also support global actions? For instance if you delete a global action from the left menu, that wouldn't necessarily mean you leave the iframe?